### PR TITLE
(add) included flights feature with skyscanner api

### PR DESCRIPTION
**Pull request by joshjnunez**
(add) flights feature
(refactor) updated schema to account for departure city and airport code
(refactor) updated code so the specific trip clicked by user is associated with each feature (hard coded values before)

### CODE REVIEW

**Review by jldaggs88 — APPROVED** [review]

File: client/components/Flights.js [hunk 1]
```diff
@@ -36,7 +37,7 @@ const Flights = ({ currentUser, currentTrip }) => {
         'https://skyscanner-skyscanner-flight-search-v1.p.rapidapi.com/apiservices/browsequotes/v1.0/US/USD/en-US/MSY-sky/BNA-sky/2020-09-10',
         {
           headers: {
-            'x-rapidapi-key': '0bab63c571mshc3c168e88ae6b44p14c63fjsn0a3bbb552a96',
```
Comment (jstrzesz) [inline client/components/Flights.js]: Rebase to remove this commit. You don't want your api key published where everyone can see it.

File: client/components/Flights.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import axios from 'axios';
+import apiKey from './config';
```
Comment (jstrzesz) [inline client/components/Flights.js:4]: place apiKey into the .env file rather than using another file

File: client/components/PurchasesForm.js [hunk 3]
```diff
@@ -40,7 +40,7 @@ const PurchasesForm = ({ savePurchase }) => {
           variant="standard"
           value={price}
           currencySymbol="$"
-     // minimumValue="0"
+          // minimumValue="0"
```
Comment (jstrzesz) [inline client/components/PurchasesForm.js:43]: remove commented out code

File: client/components/Flights.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+      });
+  }, []);
+
+  const onClickShowMeFlight = () => {
+    axios
+      .get(
+        'https://skyscanner-skyscanner-flight-search-v1.p.rapidapi.com/apiservices/browsequotes/v1.0/US/USD/en-US/MSY-sky/BNA-sky/2020-09-10',
+        {
+          headers: {
+            'x-rapidapi-key': '0bab63c571mshc3c168e88ae6b44p14c63fjsn0a3bbb552a96',
```
Comment (jstrzesz) [inline client/components/Flights.js]: Git reset to remove this commit. You don't want api key published

**Comment by jstrzesz** [comment]
DO NOT MERGE THIS PR YET, see comments

**Pull request closed without merging**